index.es.md

```diff
@@ -0,0 +1,244 @@
+
+**La Accesibilidad es esencial para desarrolladores y organizaciones que quieren crear sitios y herramientas web de calidad y no excluir personas del uso de sus productos y servicios.**
+
+
+## Qué es la Accesibilidad Web {#what}
+
+Accesibilidad Web significa que sitios web, herramientas y tecnologías están diseñadas y desarrolladas de tal manera que las personas con discapacidades pueden usarlas. Más concretamente, las personas pueden:
+
+-   percibir, comprender, navegas e interactuar con la Web
+-   contribuir a la Web
+
+La Accesibilidad web abarca todas las discapacidades que afectan al acceso a la Web, incluyendo:
+
+-   auditivas
+-   cognitivas
+-   neurológicas
+-   físicas
+-   del habla
+-   visuales
+
+La accesibilidad web también beneficia personas *sin* discapacidad, como por ejemplo:
+
+-   personas utilizando teléfonos móviles, relojes inteligentes, televisores inteligentes y otros dispositivos con pantallas pequeñas, diferentes modos de entrada, etc.
+-   personas mayores cuyas habilidades cambian con la edad
+-   personas con "discapacidades temporales", como puede ser un brazo roto o la pérdida de unas gafas
+-   personas con "limitaciones por su ubicación", como puede ser bajo la luz del sol o en un entorno donde no se puede escuchar audio
+-   personas con conexión lenta a Internet o que tienen ancho de banda limitado o costoso
+
+Puede ver un vídeo de 7 minutos con ejemplos de cómo la accesibilidad es esencial para personas con discapacidades y útil para cualquier persona en una gran variedad de situaciones:<br>
+{% include video-link.html title="Perspectivas de Accesibilidad Web <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Más información sobre Qué es Accesibilidad" class="simple aside" %}
+{:/}
+
+-   Si desea aprender más acerca de cómo las diferentes discapacidades afectan en el uso de la Web y leer sobre escenarios de personas con discapacidad usando la Web, acceda a [[Cómo las Personas con Discapacidades Utilizan la Web]](/people-use-web/).
+-   Si desea más ejemplos de beneficios para otras personas, con las WCAG como ayuda, acceda a [[Experiencias Web Compartidas: Barreras comunes para Personas Usuarias de Dispositivos Móviles y Personas con Discapacidades]](/standards-guidelines/shared-experiences/) y a [Accesibilidad Web Beneficia a Personas con y sin Discapacidades](https://www.w3.org/WAI/business-case/archive/soc#groups).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## La Accesibilidad es Importante para las Personas, Empresas y Sociedad {#important}
```

Comment:
I am leaving this comment here, but it could be applied anywhere else where appropriate.

In the Spanish language, the capital letters are not used in every word belonging to a heading. This is done in the English language, but the Spanish grammar states that in headings, titles, works of art, etc... only the first word should be capitalized (check rule 4.17 in the link below).

More info:
http://lema.rae.es/dpd/srv/search?id=BapzSnotjD6n0vZiTp
https://cvc.cervantes.es/lengua/alhabla/museo_horrores/museo_011.htm

Reply:
I added this to the [General glossary](https://github.com/w3c/translation-glossaries/blob/master/general.md#capitalization---titles-and-headings) and to the [Spanish glossary](https://github.com/w3c/translation-glossaries/blob/master/Castellano-Spanish.md#capitalization---title-and-headings).

Feel free to improve the wording there if you would like to.

Also, **please do add other things like this to the Glossaries**.

<hr>

@rumoroso I guess you want to update https://www.w3.org/WAI/standards-guidelines/mobile/es
:-)

Reply:
Great, thanks @slhenry! Will make sure to add comments like this one to the Glossary next time.